src/widgets/BugsHistory/components/BugsHistory.scss

```diff
@@ -5,4 +5,8 @@
     font-size: $font-big;
     color: $rocket-gray;
   }
+
+  .chart-container {
```

Comment:
The problem had something to do with a way how this specific chart was drawn. For example, the Jira Issues chart works okay everywhere. That's why I don't think we need this change to be applied to all widgets. How do you think?